### PR TITLE
Feature db controller

### PR DESCRIPTION
**Pull request by edsonarios**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java [hunk 1]
```diff
@@ -20,8 +21,17 @@
...
 
 import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
 
+import static org.fundacion.jala.converter.models.ProjectSQL.insertProjectData;
+import static org.fundacion.jala.converter.models.ProjectSQL.listProject;
+
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java]: This blank line should be deleted

File: src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java [hunk 2]
```diff
@@ -35,21 +45,62 @@
...
+                .map(project -> project.getPath())
+                .collect(Collectors.toList());
+        exist = resultTitle.size() > 0;
+        if (exist) {
+            filename = resultTitle.get(0);
+            storagePath = resultPath.get(0) + filename;
+        } else {
+            filename = file.getOriginalFilename();
+            storagePath = fileStorageService.uploadFile(file);
+            try {
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java:77]: Logs are missing in this try - catch

File: src/main/java/org/fundacion/jala/converter/models/Create.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+//        insertUserData("Pablo pablo", "password1", "token1");
+//        insertUserData("Elena elena", "password2", "token2");
+//        insertUserData("Edson edson", "password1", "token1");
+//        insertUserData("Cristian cristian", "password2", "token2");
+//        insertUserData("Edson", "password1", "token1");
+//        user1 = insertUserData("cris", "password2", "token2");
+//        user2 = insertUserData("ed", "password2", "token2");
+//        editUserData(19, "Cristi", "123456", "tokeeen");
+//        deleteProject(28);
+        System.out.println(listProject());
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/models/Create.java]: these comments should be removed

File: src/main/java/org/fundacion/jala/converter/models/Insert.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
+
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * <p>
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/Insert.java]: This should be deleted

File: src/main/java/org/fundacion/jala/converter/controller/AuthController.java [hunk 5]
```diff
@@ -59,7 +61,7 @@
                 .loadUserByUsername(authenticationRequest.getUsername());
 
         final String jwt = jwtTokenUtil.generateToken(userDetails);
-
+        editUserData(2,findUserById(2).getName(),findUserById(2).getPassword(),jwt);
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/controller/AuthController.java]: Maybe add a space next to the "," jwtç

File: src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java [hunk 6]
```diff
@@ -8,6 +8,7 @@
  */
 package org.fundacion.jala.converter.controller;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java]: these spaces could be removed

File: src/main/java/org/fundacion/jala/converter/controller/VideoConverterController.java [hunk 7]
```diff
@@ -17,6 +17,7 @@
 import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 
 import java.io.IOException;
+//import static org.fundacion.jala.converter.models.Insert.insertData;
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/controller/VideoConverterController.java]: You should delete comments

File: src/main/java/org/fundacion/jala/converter/models/Insert.java [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+public final class Insert {
+
+    public Insert() {
+    }
+
+    private static EntityManagerFactory emf = Persistence.createEntityManagerFactory("jalaPersistence");
+
+    /**
+     * This method is for to insert dates for database
+     * @param userName
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/Insert.java]: The javadocs in this class should specify the input params and the return value

File: src/main/java/org/fundacion/jala/converter/models/Create.java [hunk 9]
```diff
@@ -0,0 +1,43 @@
+
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/models/Create.java]: Author is missing

File: src/main/java/org/fundacion/jala/converter/models/Create.java (hunk 3 above)
Comment (edsonarios) [reply src/main/java/org/fundacion/jala/converter/models/Create.java]: Resolved

File: src/main/java/org/fundacion/jala/converter/models/Create.java [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+package org.fundacion.jala.converter.models;
+
+
+import static org.fundacion.jala.converter.models.ProjectSQL.*;
+import static org.fundacion.jala.converter.models.UserSQL.*;
+
+public final class Create {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/models/Create.java]: this class Create  seems not to be necessary

File: src/main/java/org/fundacion/jala/converter/models/ProjectSQL.java [hunk 11]
```diff
@@ -18,20 +18,20 @@
     private static EntityManagerFactory emf = Persistence.createEntityManagerFactory("jalaPersistence");
 
     /**
-     * Inserts projects in the db
-     * @param projectName String with the project name
-     * @param pathProject String with the path of the project
-     * @param type String with the type of the project
-     * @param user a User
+     * This method is for to insert dates for database
+     * @param projectName
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/ProjectSQL.java]: The javadocs in this class should specify the input params and the return value

File: src/main/java/org/fundacion/jala/converter/models/Insert.java [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+    public static void insertProjectData(final String projectName, final String pathProject, final String checksum, final User user) {
+        EntityManager manager = emf.createEntityManager();
+        manager.getTransaction().begin();
+        Project project = new Project();
+        project.setTitle(projectName);
+        project.setPath(pathProject);
+        project.setChecksum(checksum);
+        project.setUser(user);
+        manager.persist(project);
+
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/models/Insert.java]: You could delete this blank line

**Review by jpgonzales27 — APPROVED** [review]

**Review by Paola-Xaq — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/models/Project.java [hunk 13]
```diff
@@ -120,16 +120,16 @@ public void setPath(final String newPath) {
      * Obtains type of the project
      * @return
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/Project.java]: The javadoc should specify the return value

**Review by danielaSantaCruz — APPROVED** [review]

**Review by JorgeRCV — APPROVED** [review]

**Review by JRojas999 — APPROVED** [review]

**Pull request merged**